src/components/common/Footer.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import {addressIcon, emailIcon, hindfloIcon, phoneIcon}  from "./image";
```

Comment:
Apart from branch icon, use svg icons. We can try to have react icons lib and use icons from one of the lib in that

Reply:
done